web/sites/all/modules/onlinekurslabor/nm_stream/nm_stream.module

```diff
@@ -3416,7 +3552,7 @@ function nm_stream_api_v2_comment_delete_callback($comment, $token = NULL) {
 function nm_stream_api_v2_set_localhost_headers() {
   // NUR BEI ENTWICKLUNG AUF VUE.JS WIEDER AKTIVIEREN
 
-  RETURN; #NUR BEI ENTWICKLUNG AUF VUE.JS WIEDER AKTIVIEREN
+  // RETURN; #NUR BEI ENTWICKLUNG AUF VUE.JS WIEDER AKTIVIEREN
```

Comment:
hier wärs fein, wenn du vor commit immer checkst, ob das rein gerutscht is. is etwas nervig, die nächsten commits vorzuspringen und zu schauen, ob das gefixt is

Reply:
ok, ich checke, dass die Änderung nicht unnötig reinkommt. 
Ich lagere die Einstellung am besten in die lokale settings.php aus.
aber schaust du nicht den diff anhand des finalen pull requests an?

Reply:
hm ich hab immer die commits im pullrequest durchgeklickt. 
Hat auch den Hintergrund, dass es öfter besser nach vollziehbar ist, wenn peu à peu was fertiges commited wird.
Aber die andere Lösung - quasi n Merge aller commits - den hab ich noch gar nicht gesehen 🥴